### PR TITLE
Elaborate on Ore Naming Guidelines

### PR DESCRIPTION
**Pull request by phase**
I added some more examples of plugin names that are right & wrong on Ore.

### CODE REVIEW

**Comment by simon816** [comment]
Not sure on the Ore* restrictions, it could be a bit over-reaching.

File: source/ore/guidelines.rst [hunk 1]
```diff
@@ -20,8 +20,22 @@ Name
 ~~~~
 
 Your project’s submitted name should not include a version, tagline, or other description. The name should be unique and original
-and must not have a name implying it is a Sponge project (e.g. *SpongeWarp* is not allowed, *Cool Warps for Sponge* is).
-
+and must not have a name implying it is an official Sponge project.
+
+Examples of names that are **wrong**:
+- *SpongeWarp*
```
Comment (ST-DDT) [inline source/ore/guidelines.rst]: You have to add an empty line before the first entry to make this bulleted list work.

````text
text:

* entry 1
* entry 2
````

File: source/ore/guidelines.rst [hunk 2]
```diff
@@ -20,8 +20,22 @@ Name
...
+- *SpongeHome*
+- *SpongeEssentials*
+- *SpongeCloud*
+- *OreJump*
+- *OreMiner*
+- *OreSpleef*
+
+Examples of names that are **right**:
+- *CoolWarps-Sponge*
```
Comment (ST-DDT) [inline source/ore/guidelines.rst]: Same here

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Not sure on the `Ore`-Stuff either. But if thats the rule, that's OK for me.

**Comment by phase** [comment]
Removed `Ore*` restrictions (those can go case by case) and fixed the lists.

File: source/ore/guidelines.rst [hunk 3]
```diff
@@ -20,8 +20,21 @@ Name
 ~~~~
 
 Your project’s submitted name should not include a version, tagline, or other description. The name should be unique and original
-and must not have a name implying it is a Sponge project (e.g. *SpongeWarp* is not allowed, *Cool Warps for Sponge* is).
+and must not have a name implying it is an official Sponge project.
 
+Examples of names that are **wrong**:
```
Comment (dualspiral) [inline source/ore/guidelines.rst]: wrong -> not acceptable

File: source/ore/guidelines.rst [hunk 4]
```diff
@@ -20,8 +20,21 @@ Name
...
+
+- *SpongeWarp*
+- *SpongeHome*
+- *SpongeEssentials*
+- *SpongeCloud*
+
+Examples of names that are **right**:
```
Comment (dualspiral) [inline source/ore/guidelines.rst]: right -> acceptable

**Review by dualspiral — DISMISSED** [review]

**Comment by phase** [comment]
Changed language

**Review by ST-DDT — APPROVED** [review]

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/9c8240b/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/9c8240b/ore/guidelines.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/709#issuecomment-402230704), not to the direct url to the preview._

**Pull request merged**